src/cpp/src/image_generation/diffusion_pipeline.hpp

```diff
@@ -25,6 +25,16 @@ void batch_copy(ov::Tensor src, ov::Tensor dst, size_t src_batch, size_t dst_bat
     ov::Tensor(src, src_start, src_end).copy_to(ov::Tensor(dst, dst_start, dst_end));
 }
 
+ov::Tensor tensor_batch_copy(const ov::Tensor input, const size_t num_images_per_prompt, size_t batch_size_multiplier) {
```

Comment:
`batch_size_multiplier` is not used

maybe we can call this function `repeat` (like it's used in diffusers) and move to `numpy_utils` ?